app/utils/callback/lava.py

```diff
@@ -320,16 +211,17 @@ def _add_boot_log(meta, log, base_path):
 
     dir_path = os.path.join(
         base_path,
-        meta["job"],
-        meta["git_branch"],
-        meta["kernel"],
-        meta["arch"],
-        meta["defconfig_full"],
-        meta["lab_name"])
+        meta[models.JOB_KEY],
```

Comment:
nitpick
I was under the impression that data extracted from the callback json was accessed using "dict string" and data from the kCI database was accessed models.STRING.

I thought it would be clearer to know if you're accessing the dB or the callback data that way. 

But overall I'd agree on using "models.STRING" all the time if it is a 1:1 matching.

Reply:
The `meta` dictionary is not directly the JSON data received from LAVA but a dictionary created by parsing the JSON data, so these keys are not used when accessing the incoming data; the `META_DATA_MAP_*`s are used to make this key translation.  Also the code that creates the DB entries uses the same keys from the models, as in `app.utils.boot._parse_boot_from_json`.  

So I think it's fine, unless I'm missing something - in which case I agree, using the model keys in the incoming JSON data would be a bit wrong and work mostly out of luck.